src/components/FilterHeaderCategories.js

```diff
@@ -53,18 +54,19 @@ const FilterHeaderCategories = ({ selectedCategories, onFilterPress }: Props) =>
           {text.filterTitle}
         </Text>
       )}
-      <View style={styles.interestButton}>
+      <Touchable style={styles.interestButton} onPress={onFilterPress}>
         <Text type="h2" style={styles.interestButtonText}>
           {text.filterByInterest}
         </Text>
         <CategoriesFilterButton onPress={onFilterPress} />
```

Comment:
Is the `onPress` here still needed?

Reply:
It still provides a nice feedback, if you tap it directly.